src/tools/illink/src/ILLink.Shared/SharedStrings.resx

```diff
@@ -1239,4 +1239,16 @@
   <data name="TypeNameIsNotAssemblyQualifiedTitle" xml:space="preserve">
     <value>Type name is not assembly qualified.</value>
   </data>
+  <data name="ReferenceNotMarkedIsTrimmableTitle" xml:space="preserve">
+    <value>Referenced assembly is not marked as trimmable.</value>
+  </data>
+  <data name="ReferenceNotMarkedIsTrimmableMessage" xml:space="preserve">
+    <value>Referenced assembly '{0}' is not marked with AssemblyMetadataAttribute IsTrimmable=True and may not be compatible with trimming.</value>
```

Comment:
It feels like mentioning the attribute name may send people on the wrong path. We definitely don't want someone to get the idea that they should manually add the attribute.

```suggestion
    <value>Referenced assembly '{0}' is not annotated as compatible with trimming and may not be compatible.</value>
```